docs/reference/contrib/modeladmin/index.rst

```diff
@@ -117,6 +118,7 @@ to create, view, and edit ``Book`` entries.
 
     class BookAdmin(ModelAdmin):
         model = Book
+        base_url_path = 'bookadmin' # customize the URL from default to admin/bookadmin
```

Comment:
Minor nitpick. Please spell it as `customise` - comments and documentation should be in English (British) not English (US)
```suggestion
        base_url_path = 'bookadmin' # customise the URL from default to admin/bookadmin
```